kernel/kernel-pdfgenerator/pom.xml

```diff
@@ -3,8 +3,8 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>io.mosip.kernel</groupId>
-	<artifactId>kernel-pdfgenerator-itext</artifactId>
-	<version>1.3.0-beta.1</version>
+	<artifactId>kernel-pdfgenerator</artifactId>
```

Comment:
The artifactId is changed, but the library folder name is still kernel-pdfgenerator-itext!?

Reply:
its changes please cross check